patchwork/api/patch.py

```diff
@@ -78,12 +80,41 @@ class PatchConflict(APIException):
     )
 
 
+class PatchReviewIntentionSerializer(BaseHyperlinkedModelSerializer):
```

Comment:
Needs updates to reflect model name changes - if we go along with those - but otherwise this seems fine.